frontend/server/server.ts

```diff
@@ -399,13 +409,50 @@ app.all(BASEPATH  + '/' + v1beta1Prefix + '/*', proxy({
   target: apiServerAddress,
 }));
 
+const DEFAULT_FLAG = 'window.KFP_FLAGS.DEPLOYMENT=null';
+function modifyFeatureFlags(indexHtml: string): string {
+  if (DEPLOYMENT === Deployments.KUBEFLOW) {
+    return indexHtml.replace(DEFAULT_FLAG, 'window.KFP_FLAGS.MULTI_USER="KUBEFLOW"');
```

Comment:
Should this also be window.KFP_FLAGS.DEPLOYMENT?

Reply:
Oops, thanks for the catch!